Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rocker/r-ver:4.3.1
+
+ENV RENV_CONFIG_REPOS_OVERRIDE "https://cloud.r-project.org/"
+ENV RENV_PATHS_LIBRARY renv/library
+
+# Install system dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    libcurl4-openssl-dev libssl-dev libxml2-dev libgit2-dev git \
+    libudunits2-dev python3-dev python3-pip
```

Comment:
These system dependencies were also copied over [from GitLab](https://gitlab.com/ccao-data-science---modeling/models/ccao_res_avm/-/blob/master/.gitlab-ci.yml?ref_type=heads#L20).

Reply:
**suggestion**: Let's add the following as well, as they'll be needed down the road:

```
libgdal-dev libgeos-dev libproj-dev libfontconfig1-dev libharfbuzz-dev libfribidi-dev pandoc
```

Reply:
Cool, I added these in c41c3a74ca1564f33fef424e066e238bd9d0c0a5 👍🏻